apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -102,92 +110,349 @@ export default function RubricContainer({
+    },
+    {
+      element: '#tour-evidence-levels',
+      title: 'Assigning a Rubric Score',
+      intro:
+        "<p>Once you have reviewed the AI Assessment and the student's code, assign a rubric score for the learning goal.</p>",
+    },
+    {
+      element: '#tour-ai-assessment-feedback',
+      title: 'How did we do?',
+      intro:
+        '<p>Your feedback helps us make the AI Teaching Assistant more helpful to you – let us know how it did.</p>',
+    },
+  ];
+
+  // Dummy props for product tour
+  const rubricDummy = {
+    id: 1,
+    learningGoals: [
+      {
+        id: 1,
+        key: '1',
+        learningGoal: 'Variables',
+        aiEnabled: true,
+        evidenceLevels: rubric.learningGoals[0].evidenceLevels,
+      },
+    ],
+    lesson: {
+      position: 3,
+      name: 'Data Structures',
+    },
+    level: {
+      name: 'test_level',
+      position: 7,
+    },
+  };
+
+  const studentLevelInfoDummy = {
+    name: 'Grace Hopper',
+    timeSpent: 305,
+    lastAttempt: '1980-07-31T00:00:00.000Z',
+    attempts: 6,
+  };
+
+  const aiEvaluationsDummy = [
+    {
+      id: 1,
+      learning_goal_id: 1,
+      understanding: 2,
+      aiConfidencePassFail: 2,
+      evidence: '',
+    },
+  ];
+
+  const onTourStart = stepIndex => {
+    setTourButtonLabel('Next Tip');
+  };
+
+  const onTourExit = () => {
+    updateTourStatus();
+  };
+
+  const onStepChange = (nextStepIndex, nextElement) => {
+    if (nextStepIndex === 1) {
+      document.getElementById('tour-fab-bg').scrollBy(0, 1000);
+    }
+  };
+
+  const beforeStepChange = (nextStepIndex, nextElement) => {
+    if (nextStepIndex === 1) {
+      if (!open) {
+        document.getElementById('ui-floatingActionButton').click();
+      }
+    }
+    if (nextStepIndex === 2) {
+      document
+        .getElementById('tour-ai-assessment')
+        .setAttribute('z-index', 99999);
+    } else if (nextStepIndex === 4) {
+      document
+        .getElementById('tour-ai-confidence')
+        .setAttribute('z-index', 99999);
+    }
+  };
+
+  const onAfterStepChange = (newStepIndex, newElement) => {
+    if (newStepIndex === 4) {
+      document.getElementsByClassName(style.evidenceLevel)[3].click();
+    }
+  };
 
   return (
     <Draggable
       defaultPosition={{x: positionX, y: positionY}}
       onStop={onStopHandler}
     >
-      <div
-        data-testid="draggable-test-id"
-        id="draggable-id"
-        className={classnames(style.rubricContainer, {
-          [style.hiddenRubricContainer]: !open,
-        })}
-      >
-        <div className={style.rubricHeaderRedesign}>
-          <div className={style.rubricHeaderLeftSide}>
-            <img
-              src={aiBotOutlineIcon}
-              className={style.aiBotOutlineIcon}
-              alt={i18n.rubricAiHeaderText()}
-            />
-            <span>{i18n.rubricAiHeaderText()}</span>
+      {stepsEnabled ? (
```

Comment:
I am seeing a lot more duplicated code between the two branches of this conditional compared to the number of differences. my recommendation would be to DRY this up and use this conditional inline to change the behavior each place a difference is needed. otherwise it seems like we are setting ourselves up for the product tour to lag or break as other updates are made to the UI.

Reply:
![Screenshot 2024-04-04 at 2 31 57 PM](https://github.com/code-dot-org/code-dot-org/assets/8001765/180f8daa-0955-43a3-9283-566a0ead7af8)


![Screenshot 2024-04-04 at 2 32 10 PM](https://github.com/code-dot-org/code-dot-org/assets/8001765/339194d2-dc50-4673-a980-7a5d476e3847)